tests/time_tests/test_runner/conftest.py

```diff
@@ -104,6 +105,20 @@ def niter(request):
 # -------------------- CLI options --------------------
 
 
+@pytest.fixture(scope="function")
+def cl_cache_dir():
+    """Generate directory to save OpenCL cache before test run and clean up after run.
+
+    Folder `cl_cache` should be created in a directory where tests were run. In this case
+    cache will be saved correctly. This behaviour is OS independent.
+    More: https://github.com/intel/compute-runtime/blob/master/opencl/doc/FAQ.md#how-can-cl_cache-be-enabled
+    """
+    cl_cache_dir = Path(".") / "cl_cache"
+    cl_cache_dir.mkdir()
```

Comment:
mkdir will fail if the directory already exists. I suggest we check if the directory exists and either clean it up / remove before creation or print meaningful error message suggesting user remove the folder himself.

Reply:
`mkdir` will throw next exception: FileExistsError: [WinError 183] Cannot create a file when that file already exists: <path_to_dir>, so no need to add any additional error message.

And I don't think we should clean up cl_cache dir manually before run

Reply:
cl_cache dir is a pre-requisite to run all tests. I'd not expect to see this error message in the log of each test. I'd prefer test handle the directory cleanup. Or at last resort test runner check for pre-requisites before executing tests and refuse to run.

Reply:
Added clean up before run